### PR TITLE
Add deviations to metadata proto

### PR DESCRIPTION
**Pull request by prinikasn**
Add devations to metadata proto
Also updating Makefile and proto lint workflow to include the ondatra testbed.proto dependency

### CODE REVIEW

**Comment by prinikasn** [comment]
As discussed offline, I am stating with just adding one deviation to the proto at a time.

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -27,7 +27,19 @@ jobs:
...
+        # Remove any existing symlinks & empty directories
+        find "${PROTOBUF_IMPORT_DIR}" -type l -delete
+        find "${PROTOBUF_IMPORT_DIR}" -type d -empty -delete
+        # Download the required dependencies
+        go mod download
+        # Get ondatra modules we use and create required directory structure
+        go list -f "${PROTOBUF_IMPORT_DIR}/{{ .Path }}" -m github.com/openconfig/ondatra | xargs -L1 dirname | sort | uniq | xargs mkdir -p
+        # Create symlink
+        go list -f "{{ .Dir }} ${PROTOBUF_IMPORT_DIR}/{{ .Path }}" -m github.com/openconfig/ondatra | xargs -L1 -- ln -s
+        find . -name \*.proto -exec api-linter -I./${PROTOBUF_IMPORT_DIR} --disable-rule all --enable-rule core {} \+
```
Comment (greg-dennis) [inline .github/workflows/pull_request.yml:42]: put quotes around "${PROTOBUF_IMPORT_DIR}"

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -27,7 +27,19 @@ jobs:
     - name: Lint protobufs
       run: |
         go install github.com/googleapis/api-linter/cmd/api-linter@latest
-        find . -name \*.proto -exec api-linter --disable-rule all --enable-rule core {} \+
+        # Set directory to hold symlink
+        PROTOBUF_IMPORT_DIR='protobuf-import'
```
Comment (greg-dennis) [inline .github/workflows/pull_request.yml:31]: put "readonly " before PROTOBUF_IMPORT_DIR='protobuf-import'

**Review by greg-dennis — CHANGES_REQUESTED** [review]
two bash nits

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (prinikasn) [reply .github/workflows/pull_request.yml]: done

**Comment by prinikasn** [comment]
> two bash nits

done. Also updated PlatformExceptions to be a repeated field

**Review by greg-dennis — APPROVED** [review]

**Pull request merged**